cvat/apps/engine/views.py

```diff
@@ -565,6 +687,21 @@ def annotations(self, request, pk):
                     return Response(data=str(e), status=status.HTTP_400_BAD_REQUEST)
                 return Response(data)
 
+class NoPagingFilterAutoSchema(SwaggerAutoSchema):
```

Comment:
@Marishka17 
Please remove this class. It was my bad, users actually are paginated.